assets/base/i18n/en.json

```diff
@@ -928,21 +927,16 @@
   "settings_display.timezone.off": "Off",
   "settings_display.timezone.select": "Select Timezone",
   "settings.about": "About {appTitle}",
-  "settings.about.app.version": "App Version: {version} (Build {number})",
-  "settings.about.app.version.title": "App Version:",
-  "settings.about.app.version.value": "{version} (Build {number})",
-  "settings.about.button.copyInfo": "Copy info",
+  "settings.about.build": "{version} (Build {number})",
```

Comment:
These changes seem unrelated.

Reply:
is fine, this is automatically generated.